.changeset/2874.md

```diff
@@ -0,0 +1,12 @@
+---
+'@backstage/catalog-backend': minor
```

Comment:
i mean obviously it should not be minor, but i just wanted a changelog entry to get in there - is this the right way to do it at the moment while we are in alpha?